pages/PoD/IDC.md

```diff
@@ -0,0 +1,161 @@
+---
+published: true
+layout: page
+title: Office of Management and Budget Integrated Data Collection (IDC) Requirements
+permalink: PoD/IDC/
+filename: IDC.md
+id: IDC
+---
+
+Office of Management and Budget Integrated Data Collection (IDC) Requirements 
+---------------------------
+
+### May 2016 IDC Requirements for Open Data Progress, Use, and Impact
+
+**Status: Updated**
+
+> * Agencies will continue to be assessed on their Open Data Policy progress through the Project Open Data Dashboard available at: [http://labs.data.gov/dashboard/offices](http://labs.data.gov/dashboard/offices).
+* Agencies will continue to provide a public version of their Enterprise Data Inventory at
+[agency].gov/data.json and include their data.json dataset with the title “Enterprise Data
+Inventory.” If necessary, agencies will coordinate with their Freedom of Information Act (FOIA)
+offices to redact specific metadata text for “non-public” datasets only if the text is subject to a
+FOIA exemption. Agencies shall use the instructions provided by OMB at [https://project-opendata.cio.gov/redactions/](https://project-opendata.cio.gov/redactions/)
+to conduct these redactions. If redactions are present, Agencies must
+submit a non-redacted Enterprise Data Inventory JSON file to OMB in the appropriate agency
+folder here: [https://community.max.gov/x/8YamK](https://community.max.gov/x/8YamK).
+* Agencies will ensure that their customer engagement process includes a transparent two-way
+feedback mechanism. A link to an e-mail address or form by itself is not sufficient. For examples of
+sufficient tools, please see [https://project-open-data.cio.gov/engagement](https://project-open-data.cio.gov/engagement).
```

Comment:
Should these project-open-data links be updated to resources.data.gov? project-open-data.cio.gov is in sunset mode and will be gone at the end of the year.

Reply:
I think this should be safe to do so, but I'm not certain we've transitioned all relevant links over. @juliaklindpaintner what do you think?